macros/insert_into_metadata_table.sql

```diff
@@ -19,3 +21,16 @@
 
     {% do run_query(insert_into_table_query) %}
 {%- endmacro %}
+
+{% macro bigquery__insert_into_metadata_table(database_name, schema_name, table_name, content) -%}
+
+        {% set insert_into_table_query %}
+        insert into {{database_name}}.{{ schema_name }}.{{ table_name }}
+        VALUES
```

Comment:
Hi @charles-astrafy I was just playing around with your branch, and I got this query from from the statement that stores the tests (I've generalized the info yo you can play with it in your BQ instance):

```
insert into capchase-sandbox.dbt_artifacts.tests
VALUES

select
  1,
  2,
  3,
  4,
  parse_json(5),
  6,
  7,
  parse_json(8)
from values
(
  '061f741c-c350-4774-9343-c409f0518d48', 
  'test.project.test_name', 
  '2022-08-24 20:46:12.337589+00:00', 
  'test_name', 
  '["model.project.model_name"]', 
  'capchase', 
  'path/to/model.sql', 
  '[]' 
)
```
 However this query produces an error, saying that
 
 ```
 Syntax error: INSERT target cannot have an alias at [2:1]
```

Reply:
@adrpino : I think you are smth wrong because the DML that insert values in the raw table for BigQuery does not use the syntax
`insert into {{database_name}}.{{ schema_name }}.{{ table_name }} 
select ...... from values `
but it uses instead the syntax:
`      insert into {{database_name}}.{{ schema_name }}.{{ table_name }}
        VALUES` 

So it seems you are not running the code with the BigQuery adapter as it is taking the code from the default adapter.

Reply:
Hi @charles-astrafy thanks!

is there any way in which I have to enforce the BQ adapter? isn't this done by default?

Reply:
Hi @adrpino ,

The adapter that dbt will use is defined via your profiles.yaml file. You define it in the "type" parameter of the target definition within that profiles.yaml file (see screenshot in attachment).

Let me know if you still have issues.

![image](https://user-images.githubusercontent.com/100716087/187423907-1e78262d-cb0c-4d43-ba65-ac9ae6d0b167.png)